plutus-metatheory/src/Builtin.lagda.md

```diff
@@ -494,6 +494,17 @@ postulate
 ### What builtin operations should be compiled to if we compile to Haskell
 
 ```
+-- NOTE.  Many of the denotations in PlutusCore.Default.Builtins involve
```

Comment:
Can you add a name to this? Like our NOTE convention, so I can refer to it also myself